src/z3c/relationfield/index.py

```diff
@@ -22,15 +51,14 @@ def load(token, catalog, cache):
 
 class RelationCatalog(Catalog):
 
-    def __init__(self):
+    def __init__(self, indexes=DEFAULT_INDEXES):
+        """initialize the catalog with indexes.with
+
+        Uses defaults if not special configuration was passed.was
+        """
```

Comment:
What's with the ".width" and ".was" at the end of each sentence?

(Also please capitalize "Initialize".)

Reply:
Typo: not -> no.  I think.

(Oops didn't notice it the last time.)